### PR TITLE
ATED-1669 : added option to change banner colour for clients

### PR DESCRIPTION
**Pull request by avinash-anand**
@feedmypixel - as discussed, this PR is to allow the colour of attorney banner to be changed

### CODE REVIEW

File: src/main/twirl/uk/gov/hmrc/play/views/layouts/attorney_banner.scala.html [hunk 1]
```diff
@@ -13,13 +13,13 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *@
-@(name: Option[String], returnLinkUrl : String, regime: uk.gov.hmrc.play.views.helpers.AttorneyRegime, customReturnLinkText: Option[String] = None)
+@(name: Option[String], returnLinkUrl : String, regime: uk.gov.hmrc.play.views.helpers.AttorneyRegime, customReturnLinkText: Option[String] = None, bannerColor: Option[String] = None, bannerText: Option[Html] = None)
```
Comment (feedmypixel) [inline src/main/twirl/uk/gov/hmrc/play/views/layouts/attorney_banner.scala.html]: `bannerColor` should have a more generic nam. I would suggest `bannerClasses`

File: src/main/twirl/uk/gov/hmrc/play/views/layouts/attorney_banner.scala.html [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
 
 <div class=" grid-wrapper attorneyContainer">
-    <div class="attorneyBanner grid" id="attorneyBanner">
+    <div class='attorneyBanner grid @bannerColor.getOrElse("")' id="attorneyBanner">
         <div class="grid grid-2-3">
             <div class="inner-block">
-                @Html(Messages(s"${regime.messagePrefix}attorney.banner.user", name.getOrElse(Messages("attorney.banner.nan"))))
+                @bannerText.getOrElse(Html(Messages(s"${regime.messagePrefix}attorney.banner.user", name.getOrElse(Messages("attorney.banner.nan")))))
```
Comment (feedmypixel) [inline src/main/twirl/uk/gov/hmrc/play/views/layouts/attorney_banner.scala.html]: `bannerText` is HTML, maybe better of being called `bannerContent`

Comment (feedmypixel) [inline src/main/twirl/uk/gov/hmrc/play/views/layouts/attorney_banner.scala.html]: this feels a little messy, maybe abstract it into a variable so it is easier to read?

```
@attorneyContent = @{Html(Messages(s"${regime.messagePrefix}attorney.banner.user", name.getOrElse(Messages("attorney.banner.nan")))){
```

then:

```
@bannerContent.getOrElse(attorneyContent)
```

**Comment by feedmypixel** [comment]
LGTM 👍

**Comment by AndyNewton** [comment]
@avinash-anand  Looking at this PR on behalf of #team-bta as we are the nominal owners of the project.

Please can you confirm what testing you have performed of this change?  Thanks.

**Comment by AndyNewton** [comment]
@avinash-anand  Please describe how this change has been tested.

**Comment by avinash-anand** [comment]
@AndyNewton - I have tested it locally with the changes. below is the screenshot.
This PR helps change the banner text as well as colour.
selection_005

**Pull request merged**